tests/test_assert_greater_than.m

```diff
@@ -0,0 +1,20 @@
+function test_suite=test_assert_greater_than()
+    try % assignment of 'localfunctions' is necessary in Matlab >= 2016
+        test_functions=localfunctions();
+    catch % no problem; early Matlab versions can use initTestSuite fine
+    end
+    initTestSuite;
+
+function test_assert_greater_than_exceptions
+    assertExceptionThrown(@()assertGreaterThan([1],'a'), ...
+                          'assertGreaterThan:classNotEqual');
+    assertExceptionThrown(@()assertGreaterThan([1 2],[1,2,3]));
+    assertExceptionThrown(@()assertGreaterThan(1,1), ...
+                          'assertGreaterThan:notGreaterThan');
+    assertExceptionThrown(@()assertGreaterThan([2,3,3],[1,2,3]), ...
```

Comment:
Can you please add tests with (1) scalar NaN compared with other NaN, scalar NaN compared with scalar non-NaN, (3) 4-dimensional NaN compared with 4-dimensional NaN, e.g. zeros(2,2,2,2)+NaN; (4) 4-dimensional arrays of the same size but different non-NaN values? Also for the less_than tests please.